test/auth.js

```diff
@@ -0,0 +1,22 @@
+export const auth = {
+  strategy: 'jwt_token',
+  credentials: {
+    user: {
+      aud: '7ae2d9fd-fe1e-47eb-9821-b6f6cd3ceba1',
+      iss: 'https://login.microsoftonline.com/2f504113-6b64-43f2-ade9-242e05780017/v2.0',
+      iat: 1716455257,
+      nbf: 1716455257,
+      exp: 1716459157,
+      aio: 'AUQAu/8WAAAAn0nUyr8POzehuskW6o0b6ANfDsrTc72EjixoJW1mRQI7uyqNRmO7MK4biieLUzJ7Y9ZUsvJ0oM4We6IH2sLb7A==',
+      groups: ['5649296f-2156-4d61-8ac3-349276438ef8'],
+      name: 'Enrique Chase (Defra)',
+      oid: '396e84b4-1cbd-40d0-af83-857be2aaefa7',
+      preferred_username: 'Enrique.Chase@defradev.onmicrosoft.com',
```

Comment:
See the `groups` and `preferred_username` properties?

These were the non-standard fields we adjusted the types for